extensions/smallrye-graphql/deployment/src/main/resources/dev-templates/embedded.html

```diff
@@ -1,7 +1,7 @@
-<a href="{config:property('quarkus.smallrye-graphql.root-path')}/schema.graphql" class="badge badge-light">
+<a href="{config:http-path('quarkus.smallrye-graphql.root-path')}/schema.graphql" class="badge badge-light">
   <i class="fa fa-project-diagram fa-fw"></i>
   GraphQL Schema</a>
 <br>
-<a href="{frameworkRootPath}{config:property('quarkus.smallrye-graphql.ui.root-path')}/" class="badge badge-light">
+<a href="{config:http-path('quarkus.smallrye-graphql.ui.root-path')}/" class="badge badge-light">
```

Comment:
Does this mean it hangs off the HTTP root and not the framework root?

I'm not sure what `config:http-path()` converts to so I could be missing it

Reply:
cool, thanks

Reply:
This was _added_ in this PR. See DevConsoleProcessor line 325-ish